### PR TITLE
Updates to Readme.md and fixes to cdm.finos.org

### PR DESCRIPTION
**Pull request by eteridvalishvili**
# Release Notes for PR https://github.com/finos/common-domain-model/pull/2160

#### Updated Readme.md and fixed issues in cdm.finos.org.
- Updated Governance.md and Code of Conduct with links replaced by Terms of Reference PDFs, now located in the "docs" directory.
- Added relative links in the readme to ensure they work seamlessly after merging.
- Included Common Domain Model in the Open Source at FINOS PDF in the Readme.md section.
- Added a roadmap section for upcoming releases.
- Edited the "Getting Involved" section.
- Updated icons for cdm.finos.org.
- Updated the banner for cdm.finos.org.
- Refined the text to explain what Common Domain Model (CDM) is and its benefits.
- Included a PDF document explaining what CDM is.

### CODE REVIEW

File: README.md
Comment (lolabeis) [inline README.md:33]: These links reference your own GitHub fork. The files should be positioned in the main repo and the links updated

**Review by lolabeis — CHANGES_REQUESTED** [review]

File: README.md
Comment (eteridvalishvili) [reply README.md]: Hi @lolabeis I think everything is ready for merging. I've replaced the links with the Terms of Reference PDFs which I added in the "docs" directory and updated the readme with relative links to ensure they work after merging, without requiring any additional changes.

As for the structure products WG, since it doesn't have one yet, I've left it linked to the community calendar for now.

Let me know if you have any questions.

Comment (eteridvalishvili) [reply README.md]: Hi @lolabeis Just wanted to follow up and see if this is okay to be merged.

**Comment by eteridvalishvili** [comment]
Hi @minesh-s-patel and @lolabeis  I've made additions to the PR with edits and fixes to [cdm.finos.org.](cdm.finos.org) Can we proceed with merging the PR? We are hoping to redirect the [landing page](https://www.finos.org/common-domain-model) to [cdm.finos.org.](cdm.finos.org). Thank you!

**Comment by maoo** [comment]
Looking at Netlify logs, it is complaining about the change on website/docusaurus.config.js , as a broken link to `docs/what_is_the_cdm.pdf` was introduced.

As you can see, this PR adds PDF files into the `docs` folder, which is intended to only host Markdown files.

All static files should go into the `website/static` folder, like images (see `img/` prefix).

I would suggest that non-trivial changes to the website are tested locally:
```
git clone git@github.com:finos/common-domain-model.git
cd common-domain-model/website
npm install
npm build
```

The `npm run build` will run a link scanner and raise errors when finding a broken link.

To check the website locally, run `npm run start`.

@eteridvalishvili - please work with @TheJuanAndOnly99 , who can quickly support you through the local setup steps and address the issue at hand.

Thanks!

File: README.md
Comment (maoo) [reply README.md]: @eteridvalishvili - the issue is still there. you can simply search on `README.md` for `eteri` and you'll spot the issues mentioned by Leo.

CC @TheJuanAndOnly99

File: RELEASE.md
Comment (maoo) [inline RELEASE.md:1]: @eteridvalishvili @TheJuanAndOnly99 - not sure why this file needs to be changed at all.

the `RELEASE.md` file hosts content related with CDM releases, and it's only touched a) by CDM maintainers and b) during a release process.

Please revert any change to this file. Thank you!

**Review by maoo — CHANGES_REQUESTED** [review]
Almost done! we just need to revert changes to RELEASE.md

File: README.md
Comment (maoo) [reply README.md]: Confirming that this is fixed now!

File: RELEASE.md
Comment (eteridvalishvili) [reply RELEASE.md:1]: I reverted the changes!

**Review by maoo — APPROVED** [review]

**Review by maoo — DISMISSED** [review]
LGTM

**Review by tomhealey-icma — APPROVED** [review]

**Pull request merged**
